### ISSUE TITLE
Remove memory usage logging

### ISSUE THREAD

**Issue by jwodder** [dandi/dandidav#154]
As noted [here](https://github.com/dandi/dandidav/issues/54#issuecomment-2152921964), the functionality for logging memory usage before & after each request causes request processing to slow down by an order of magnitude.  This functionality should be removed (or at least put behind a non-default feature) once #118 is resolved.

**Comment by jwodder** [comment]
@yarikoptic Can I do this now that #118 is resolved?

**Comment by yarikoptic** [comment]
yeah, let's do that I guess since we assume it was addressed.  There is no easy way to make it "optional" (compile or run time) while avoiding impact?  This way we would be able to quickly turn it on to troubleshoot if it would come needed

**Comment by jwodder** [comment]
@yarikoptic It could be made optional at either runtime or compile time; which do you prefer?

**Comment by yarikoptic** [comment]
it is up to you since you would be the most likely "user" ;-)  "runtime" sounds better to me, if doesn't entail any runtime performance hit.
